### ISSUE TITLE
Do not report MA0028 for concatenation of constant strings and chars

### ISSUE THREAD

**Issue by cbersch** [meziantou/Meziantou.Analyzer#1528]
### Version of the Meziantou.Analyzer NuGet package

3.0.259

### Rule Identifier

MA0028

### Target Framework

net10

### C# Language version

C# 14.0

### .NET SDK version

10.0.401

### Description

String concatenation of constant strings and chars seems to be merged by the compiler.

### Reproduction Steps

Minimal code:

```c#
public sealed class MyClass
{
    public const char Const = 'C';

    public string Property => new StringBuilder().Append(" " + Const + " ").ToString();
}
```

This raises a MA0028 although it shouldn't.
